easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-001-system.yml

```diff
@@ -0,0 +1,4 @@
+easyconfigs:
+  - EasyBuild-4.9.2.eb:
+      options:
+        from-pr: 20801
```

Comment:
Use https://github.com/easybuilders/easybuild-easyconfigs/pull/20818 instead, and use EasyBuild v4.9.1 to install EasyBuild v4.9.2

Reply:
done in https://github.com/EESSI/software-layer/pull/604/commits/158f0eb96d2027a51bb8d5ad8ae347405790c8d4